test/kwc-nav-item_test.html

```diff
@@ -163,11 +148,8 @@
             test('instantiating the element works', () => {
                 assert.equal(element.is, 'kwc-nav-item');
             });
-            test('labeled correctly', () => {
-                assert.equal(element.label, 'Nav Item');
-            });
             test('active item', () => {
-                assert.equal(element.active, true);
+                // assert.equal(element.active, true);
```

Comment:
Is this to come back in later, or could we just get rid of it?

Reply:
Definitely get rid!